src/styles/main.scss

```diff
@@ -1,3 +1,265 @@
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link:hover {
+    color: $blue;
+  }
+}
+
+
+.is-active {
+  color: $blue;
+}
+
+
+.is-active::after {
+  content: "";
+  width: 100%;
+  height: 4px;
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  border-radius: 8px;
+  background-color: $blue;
+  cursor: default;
+}
+
+
+/* Main-page */
+.page {
+  display: flex;
+  justify-content: center;
+
+  &__container {
+    display: grid;
+    padding: 50px 40px;
+    justify-items: center;
+    gap: 46px;
+    column-gap: 48px;
+    grid-template-columns: repeat($column-count-small-width, 1fr);
+  }
+}
+
+
+@media (min-width: 488px) {
+  .page__container {
+    grid-template-columns: repeat($column-count-488width, 1fr);
+  }
+}
+
+@media (min-width: 768px) {
+  .page__container {
+    grid-template-columns: repeat($column-count-768width, 1fr);
+  }
+}
+
+@media (min-width: 1024px) {
+  .page__container {
+    grid-template-columns: repeat($column-count-1024width, 1fr);
+  }
+}
+
+
+.card {
+  max-width: 200px;
+  border: 1px solid #F3F3F3;
+  transition: all 0.3s;
+
+  &__container {
+    padding: 32px 16px 16px;
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__photo {
+    display: flex;
+    justify-content: center;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__photo--margin {
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: $big-line-height;
+  }
+
+  &__title--margin {
+    margin-bottom: 4px;
+  }
+
+  &__subtitle--margin {
+    margin-bottom: $default-margin;
+  }
+  &__subtitle {
+    color: $secondary-accent;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: $default-line-height;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+
+    .card__title {
+      color:  #34568b;
+    }
+  }
+
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    height: 40px;
+    padding: 12px 70px;
+    color: $white;
+    font-size: 14px;
+    line-height: $default-line-height;
+    font-weight: 700;
+    text-transform: uppercase;
+    text-decoration: none;
+    background-color: $blue;
+    border-radius: 5px;
+    transition: background-color 0.3s linear ,color 0.3s linear;
+    border: 1px solid $blue;
+  }
+
+  &__button:hover {
+    background-color: $white;
+    color: $blue;
+  }
+}
+
+.feedback {
+  display: flex;
+  justify-content: space-between;
+
+
+  &__review {
+    display: flex;
+    color: $main-accent;
+    font-size: 10px;
+    font-weight: 400;
+    align-self: center;
+  }
+
+  &--margin {
+    margin-bottom: 24px;
+  }
+}
+
+
+.stars {
+  display: flex;
+  gap: 4px;
+
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:not(:last-child) {
+    background-image: url(../images/star-active.svg);
+  }
+}
+
+.stars__star--5 {
+  background-image: url(../images/star.svg);
+}
```

Comment:
fix it then, and implement styles for all modifiers, not only for 4 active stars